### PR TITLE
Devo_v2 initial version with enhanced functionality

### PR DESCRIPTION
**Pull request by ansfan**
Features now include fetching alerts, running multi-table queries, writing back to Devo tables. Please refer to README.md for all features. Refer to previous PR in related issues as 

## Status
Ready

## Related Issues
https://github.com/demisto/content/pull/4419

## Description
A new and updated Devo_v2 integration with enhanced functionality. Uses our new APIs and offers automated incident creation of alerts generated in Devo.

## Screenshots
image
image

## Required version of Demisto
5.0.0+

## Does it break backward compatibility?
   - No

## Must have
- [X] Tests
- [X] Documentation (with link to it)
- [X] Code Review

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.

## Additional changes
Completely new integration

## Technical writer review
Mention and link to the files that require a technical writer review.
- [X] [YAML file](https://github.com/devods/content/blob/54b778ed622e92d5c515faaf3a4a95aef73a92cf/Integrations/Devo_v2/Devo_v2.yml)
- [X] [CHANGELOG](link)

### CODE REVIEW

**Comment by yaakovi** [comment]
additional points from the demo:
* switch json parameter to "long text"
* error handling for test button, add a proper error message:
  * invalid json for writer (if json.loads fails)
  * wrong credentials (couldn't login)
  * in case of fetch-incident, try using the given query

File: Integrations/Devo_v2/Devo_v2.yml [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    - name: to
+      description: End datetime for specified query. Unix timestamp in seconds expected
+        (Decimal milliseconds okay)
+    - name: writeToContext
+      description: Whether to write results to context or not
+      defaultValue: "true"
+    outputs:
+    - contextPath: Devo.QueryResults
+      description: List of dictionary of results
+      type: unknown
```
[the remark this answers is not in the document]
Comment (ansfan) [reply Integrations/Devo_v2/Devo_v2.yml]: The structure will be wholly dependent on the query that the user gives and the tables that they are querying.

File: Integrations/Devo_v2/Devo_v2.yml [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    outputs:
+    - contextPath: Devo.MultiResults
+      description: List of dictionary results
+    description: Queries multiple tables for a given token and returns relevant results
+  - name: devo-write-to-table
+    arguments:
+    - name: tableName
+      required: true
+      description: Table name to write to
+    - name: records
```
[the remark this answers is not in the document]
Comment (ansfan) [reply Integrations/Devo_v2/Devo_v2.yml]: No tableName is just a string

File: Integrations/Devo_v2/Devo_v2.yml [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+    - name: query
+      required: true
+      description: A LINQ Query to run
+    - name: from
+      required: true
+      description: Start datetime for specified query. Unix timestamp in seconds expected
+        (Decimal milliseconds okay)
+    - name: to
+      description: End datetime for specified query. Unix timestamp in seconds expected
+        (Decimal milliseconds okay)
```
[the remark this answers is not in the document]
Comment (ansfan) [reply Integrations/Devo_v2/Devo_v2.yml]: https://github.com/demisto/content/pull/5233/commits/c907e3a5aa012335266e2edcaa005a550b8d24ca

File: Integrations/Devo_v2/Devo_v2.yml [hunk 4]
```diff
@@ -49,12 +44,22 @@ configuration:
...
   required: false
-- display: Deduplication parameters JSON if required. SEE README
+- display: Deduplication parameters JSON if required.
   name: fetch_incidents_deduplication
   defaultvalue: ""
-  type: 0
+  type: 12
+  required: false
+- display: Allow insecure Devo endpoint connection
+  name: allow_insecure
```
[the remark this answers is not in the document]
Comment (ansfan) [reply Integrations/Devo_v2/Devo_v2.yml]: https://github.com/demisto/content/pull/5233/commits/68c6f006f61d77725f0454e6100350d8289ec333
https://github.com/demisto/content/pull/5233/commits/f8edbd55e837e001e49e48eedad9ee2bf990f82b

**Comment by yaakovi** [comment]
hi @kirbles19 ,
please go over the PR files, it's a new integration.

**Comment by kirbles19** [comment]
@yaakovi Done.

**Pull request merged**